tests/Application/config/packages/dev/monolog.yaml

```diff
@@ -7,3 +7,8 @@ monolog:
         firephp:
             type: firephp
             level: info
+        sylius_tpay:
+            type: stream
+            path: "%kernel.logs_dir%/sylius_tpay_%kernel.environment%.log"
+            level: debug
+            channels: [sylius_tpay]
```

Comment:
Such a config should be added to documentation to make it work or some how added to main configs